log/clilog.go

```diff
@@ -117,7 +117,7 @@ func (log *CliLog) writeLog(verbosity int, messages ...string) {
 
 	case VERBOSITY_MESSAGE:
 		prefix := log.stack[len(log.stack)-1] + ": "
-		if length := utf8.RuneCountInString(prefix); length < 25 {
+		if length := utf8.RuneCountInString(prefix); length < 15 {
```

Comment:
This change is a fix for a runtime error.  This integer needs to correspond to the below integer, or you can get runtime errors.  @TODO it needs to be moved to a constant.